Resultpage/resultpage.css

```diff
@@ -0,0 +1,78 @@
+.result-page {
+  margin: 6rem 2rem 2rem 2rem;
```

Comment:
suggestion- you can make utility classes for this

Reply:
Yes.